### PR TITLE
Cucumber, RSpec rake tasks from their libraries

### PR DESCRIPTION
**Pull request by olleolleolle**
Use the RakeTask classes from Cucumber and RSpec to configure their Rake tasks in the `test` namespace.

- these tasks all have the same names as before
- and the same settings

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -31,20 +31,20 @@ task :rubocop do
...
-  desc 'Run cucumber tests'
-  task :cucumber do
-    sh 'bundle exec cucumber'
-  end
+  Cucumber::Rake::Task.new
 
-  desc 'Run cucumber tests which are "WORK IN PROGRESS" and are allowed to fail'
-  task :cucumber_wip do
-    sh 'bundle exec cucumber -p wip'
+  Cucumber::Rake::Task.new(:cucumber_wip, 'Run cucumber tests which are "WORK IN PROGRESS" and are allowed to fail') do |t|
```
Comment (junaruga) [inline Rakefile]: Right now this project's max line length is 169. (I do not know why).
But I think considering default line max length 80 is better. (inserting line break)
https://github.com/cucumber/aruba/blob/master/.rubocop.yml#L39

File: Rakefile [hunk 2]
```diff
@@ -31,20 +31,20 @@ task :rubocop do
...
   end
 
-  desc 'Run rspec tests'
-  task :rspec do
-    sh 'bundle exec rspec'
+  begin
+    require 'rspec/core/rake_task'
+    desc 'Run rspec tests'
+    RSpec::Core::RakeTask.new(:rspec)
+  rescue LoadError
```
Comment (junaruga) [inline Rakefile]: This part causes rubocop error. 

https://travis-ci.org/cucumber/aruba/jobs/259067798
> Rakefile:47:3: W: Lint/HandleExceptions: Do not suppress exceptions.

**Comment by junaruga** [comment]
Hi @olleolleolle 
Sorry I merged below PR that conflicts with your PR
https://github.com/cucumber/aruba/pull/468

Could you rebase on latest master branch, and add "-f progress" to your `Cucumber::Rake::Task`?

Now you can try your local test easily by this PR.
https://github.com/cucumber/aruba/pull/467

Thank you.

**Comment by aslakhellesoy** [comment]
Hi @olleolleolle,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for we ask you to:

* ✅ Please continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by junaruga** [comment]
It looks good to me.
Thanks!

**Pull request merged**